### PR TITLE
Tahoma Platform

### PR DESCRIPTION
**Pull request by philklei**
**Description:**

Success of #3002.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
https://github.com/home-assistant/home-assistant/pull/10652

### CODE REVIEW

File: source/_components/tahoma.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+```yaml
+# Example configuration.yaml entry
+tahoma:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+  exclude: [BridgeHUEComponent, HueLampHUEComponent, PodComponent]
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/tahoma.markdown]: Please use the configuration tags as described in our documentation guidelines:

For more information please visit:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/tahoma.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
+---
+
+
+The `Tahoma` component platform is used as an interface to the [tahomalink.com](https://www.tahomalink.com) website. It adds actually covers and the sun sensor from tahoma plattform.
```
Comment (frenck) [inline source/_components/tahoma.markdown]: Typo: `plattform` -> `platform`.

The last sentence is not quite right. I would suggest removing it or maybe reword it?
Suggestion: `It adds covers and the sun sensor from tahoma platform.`

File: source/_components/tahoma.markdown (hunk 1 above)
Comment (philklei) [reply source/_components/tahoma.markdown]: @frenck updated!

**Comment by frenck** [comment]
@philklei 🏅 
Can be merged as soon as the parent PR gets merged.
Note: Version needs to be updated as soon as the parent gets merged.

File: source/_components/tahoma.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+```yaml
+# Example configuration.yaml entry
+tahoma:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+  exclude: [BridgeHUEComponent, HueLampHUEComponent, PodComponent]
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/tahoma.markdown]: You can remove this header since you have implemented the configuration tag instead.

File: source/_components/tahoma.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+title: "Tahoma"
+description: "Instructions on how to integrate Somfy Tahoma devices with Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
```
Comment (MartinHjelmare) [inline source/_components/tahoma.markdown]: 0.59

File: source/_components/sensor.tahoma.markdown [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+title: "Tahoma Sensor"
+description: "Instructions how to integrate Tahoma sensors into Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.41
```
Comment (MartinHjelmare) [inline source/_components/sensor.tahoma.markdown]: 0.59

File: source/_components/cover.tahoma.markdown [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+title: "Tahoma Cover"
+description: "Instructions how to integrate Tahoma covers into Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.41
```
Comment (MartinHjelmare) [inline source/_components/cover.tahoma.markdown]: 0.59

**Comment by philklei** [comment]
@MartinHjelmare update files

File: source/_components/cover.tahoma.markdown [hunk 7]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Tahoma Cover"
+description: "Instructions how to integrate Tahoma covers into Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
```
Comment (MartinHjelmare) [inline source/_components/cover.tahoma.markdown]: There doesn't seem to be a logo named `tahoma.png`. Am I wrong?

File: source/_components/sensor.tahoma.markdown [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Tahoma Sensor"
+description: "Instructions how to integrate Tahoma sensors into Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
```
Comment (MartinHjelmare) [inline source/_components/sensor.tahoma.markdown]: Same question as above.

File: source/_components/cover.tahoma.markdown (hunk 7 above)
Comment (frenck) [reply source/_components/cover.tahoma.markdown]: You are absolutely right!
@philklei ?

**Comment by frenck** [comment]
@philklei, thank you for updating the PR! 🏅
